src/mesh/generated/meshtastic/mesh.pb.h

```diff
@@ -109,7 +109,7 @@ typedef enum _meshtastic_HardwareModel {
     meshtastic_HardwareModel_NRF52840_PCA10059 = 40,
```

Comment:
These files are automatically generated. You probably want to propose a change to 
https://github.com/meshtastic/protobufs

Reply:
Thanks, I'll do that.